libs/sdk-ui-tests-e2e/cypress/tools/apiRequest/LocalizationApi.ts

```diff
@@ -0,0 +1,92 @@
+// (C) 2022-2023 GoodData Corporation
+import { getHost, getTigerAuthToken, getProjectId } from "../../support/constants";
+
+class LocalizationApi {
+    localeSettingViaAPI = (value: string, level: string) => {
+        let url: string;
+        if (level === "userSetting") {
+            cy.request({
+                method: "GET",
+                url: `${getHost()}/api/v1/profile`,
```

Comment:
Is it possible to use `api-client-tiger` for these requests instead?

Reply:
I haven't tried using `api-client-tiger` yet. Do you have any existing test which use it? please send it to me

Reply:
I think it should be possible to process all these requests with usage of `api-client-tiger`, not sure if it's used anywhere in the tests now. Or in the best case, can we use normal `sdk-backend-tiger` instance and its methods? There is setLocale method if I'm correct.

Reply:
<img width="802" alt="Screen Shot 2023-02-28 at 17 09 05" src="https://user-images.githubusercontent.com/82133226/221822971-73d736a2-bb7b-47c2-9d7f-3af58fa91d55.png">

I tried 
`import sdk from "@gooddata/api-client-tiger";
sdk.entities....`
but I can't find setLocale or updateLocale or anything related to locale (I can find theme/colorpalette,... but with locale I can't)

Reply:
setLocale method is in `sdk-backend-tiger`

Reply:
IMO, i don't think `sdk-backend-tiger` should be used in this test. 
This test is based on real user actions, (refer to https://www.gooddata.com/developers/cloud-native/doc/cloud/customize-appearance/change-display-language/), verify locale is applied if sending API requests directly as the document mentioned